source-code/sdk-js/src/adapter-sveltekit/ast-transformation/transforms/*.svelte.ts

```diff
@@ -38,14 +44,24 @@ export const transformSvelte = async (config: TransformConfig, code: string): Pr
 		? `import { getRuntimeFromContext } from "@inlang/sdk-js/adapter-sveltekit/client/not-reactive";`
```

Comment:
comment for line 24: is `inn` the alias if `i` is already declared somewhere else?
If true, than what do we do if `inn` also exists? We can tackle that later, just add a TODO for now